src/auro-menu.js

```diff
@@ -37,10 +37,10 @@ class AuroMenu extends LitElement {
     ];
   }
 
-  connectedCallback() {
-    super.connectedCallback();
+  firstUpdated() {
     this.addEventListener('keydown', this.handleKeyDown);
```

Comment:
This interaction is not consistent with the native HTML select option menu nor examples from shoelace or adobe. 

If auro-combobox requires this interaction because of an implementation issue, then I would suggest that we make this a prop feature in the next series of updates per the new a11y model.